_posts/2022-09-12-tenth-birthday-celebration-recap.md

```diff
@@ -3,14 +3,13 @@ layout: post
 title: "Trino's tenth birthday celebration recap"
 author: "Brian Olsen"
 excerpt_separator: <!--more-->
+image: /assets/blog/trino-tenth-birthday/creators.jpeg
 ---
 
 What an exciting month we had in August! August marked the ten-year birthday of
```

Comment:
We need a little bit of whitespace or a div with a small margin or something before this line, it's currently awkwardly close to the image on the blog page itself with no gap.

<img width="883" alt="image" src="https://user-images.githubusercontent.com/104291808/189962609-5d9af129-ae45-49fc-99fb-b07a9ce86434.png">